Mortgage lenders are interested in determining borrower and loan factors that may lead to delinquency or foreclosure. In the file $\textit{lasvegas.dat}$ are 1,000 observations on mortgages for single family homes in Las Vegas, Nevada during 2008. The variable of interest is $\textit{DELINQUENT}$, an indicator variable $= 1$ if the borrower missed at least three payments ($90+$ days late), but $0$ otherwise. Explanatory variables: are $\textit{LVR}=$  the ratio of the loan amount to the value of the property; $\textit{REF}=1$ if purpose of the loan was a ‘‘refinance’’ and $=0$ if loan was for a purchase; $\textit{INSUR}=1$ if mortgage carries mortgage insurance, $0$ otherwise; $\textit{RATE}=$ initial interest rate of the mort- gage; $\textit{AMOUNT}$  dollar value of mortgage (in $\$100,000$); $\textit{CREDIT}=$ credit score, $\textit{TERM}=$ number of years between disbursement of the loan and the date it is expected to be fully repaid, $\textit{ARM}=1$  if mortgage has an adjustable rate, and $=0$ if mortgage has a fixed rate.

Estimate the linear probability (regression) model explaining DELINQUENT as a function of the remaining variables. Use White robust standard errors. Are the signs of the estimated coefficients reasonable?

In [1]:
clear all
use https://www.stata.com/data/s4poe4/lasvegas.dta
reg delinquent lvr ref insur rate amount credit term arm






      Source |       SS           df       MS      Number of obs   =     1,000
-------------+----------------------------------   F(8, 991)       =     62.77
       Model |  53.6062717         8  6.70078397   Prob > F        =    0.0000
    Residual |  105.792728       991   .10675351   R-squared       =    0.3363
-------------+----------------------------------   Adj R-squared   =    0.3309
       Total |     159.399       999  .159558559   Root MSE        =    .32673

------------------------------------------------------------------------------
  delinquent |      Coef.   Std. Err.      t    P>|t|     [95% Conf. Interval]
-------------+----------------------------------------------------------------
         lvr |   .0016239   .0007846     2.07   0.039     .0000843    .0031634
         ref |  -.0593237   .0238299    -2.49   0.013    -.1060865   -.0125609
       insur |  -.4815849   .0236365   -20.37   0.000    -.5279683   -.4352015
        rate |   .0343761   .0085999     4.00  

(b) Use probit to estimate the model in (a). Are the signs and significance of the estimated coefficients the same as for the linear probability model?

In [2]:
probit delinquent lvr ref insur rate amount credit term arm


Iteration 0:   log likelihood =   -499.013  
Iteration 1:   log likelihood = -338.38904  
Iteration 2:   log likelihood = -332.81547  
Iteration 3:   log likelihood = -332.79661  
Iteration 4:   log likelihood = -332.79661  

Probit regression                               Number of obs     =      1,000
                                                LR chi2(8)        =     332.43
                                                Prob > chi2       =     0.0000
Log likelihood = -332.79661                     Pseudo R2         =     0.3331

------------------------------------------------------------------------------
  delinquent |      Coef.   Std. Err.      z    P>|z|     [95% Conf. Interval]
-------------+----------------------------------------------------------------
         lvr |   .0076007   .0045911     1.66   0.098    -.0013977    .0165991
         ref |  -.2884561   .1259446    -2.29   0.022    -.5353029   -.0416092
       insur |  -1.772714   .1158088   -15.31   0.000    -1.9

(c) Compute the predicted value of DELINQUENT for the 500th and 1,000th observations using both the linear probability model and the probit model. Interpret the values.

(d) Construct a histogram of CREDIT. Using both linear probability and probit models, calculate the probability of delinquency for CREDIT 1⁄4 500, 600, and 700 for a loan of $250,000 (AMOUNT 1⁄4 2.5). For the other variables, loan to value ratio (LVR) is 80%, initial interest rate is 8%, indicator variables take the value one, and TERM 1⁄4 30. Discuss similarities and differences among the predicted probabilities from the two models.

Compute the marginal effect of CREDIT on the probability of delinquency for CREDIT 1⁄4 500, 600, and 700, given that the other explanatory variables take the values in (d). Discuss the interpretation of the marginal effect.


Using a data introduced in one of the past pre-class work questions, check whether **margins **command in Stata correctly estimates the marginal effect of an interaction term in a nonlinear model. Note: Use a probit model and interact two continuous variables.